cmd/ocm/create/cluster/cmd.go

```diff
@@ -1054,7 +1054,7 @@ func promptExistingAWSVPC(fs *pflag.FlagSet, connection *sdk.Connection) error {
 
 func wasGCPNetworkReceived() bool {
 	return args.existingVPC.VPCName != "" && args.existingVPC.ControlPlaneSubnet != "" &&
-		args.existingVPC.ComputeSubnet != ""
+		args.existingVPC.ComputeSubnet != "" && args.existingVPC.VPCProjectID != ""
```

Comment:
@JakobGray   vpcProjectID is optional.   Incase of a Non-XPN existing VPC we wouldnt set VPCProjectID.  I dont think we should be doing a "&&" here